babel/core.py

```diff
@@ -885,6 +885,41 @@ def ordinal_form(self):
         """
         return self._data.get('ordinal_form', _default_plural_rule)
 
+    @property
+    def measurement_systems(self):
+        """Localized names for various measurement systems.
+
+        >>> Locale('fr', 'FR').measurement_systems['US']
+        u'am\\xe9ricain'
+        >>> Locale('en', 'US').measurement_systems['US']
+        u'US'
+
+        """
+        return self._data['measurement_systems']
+
+    @property
+    def character_order(self):
+        """The text direction for the language.
+
+        >>> Locale('de', 'DE').character_order
+        'left-to-right'
+        >>> Locale('ar', 'SA').character_order
+        'right-to-left'
+        """
+        return self._data['character_order']
+
+    @property
```

Comment:
Well, `character_order` (well, `characterOrder`, but we're snakes, not camels) is the CLDR name, so I'd like to keep that as-is. Likewise, `direction` is the CSS property (though I think it's better spelled as `text_direction` in Babel, just to be explicit).